### PR TITLE
Update discovery flow for edge-authorizers

### PR DESCRIPTION
**Pull request by carabasdaniel**

### CODE REVIEW

File: docs/edge-authorizers/discovery-flow.mdx [hunk 1]
```diff
@@ -128,8 +127,8 @@ The expected response should be:
             "max_delay_seconds": 120,
             "min_delay_seconds": 60
           },
-          "resource": "<POLICY_ID>/bundle.tar.gz",
-          "service": "aserto-bundler",
+          "resource": "<TENANT>/<POLICY>/blobs/sha256:<DIGEST>",
```
Comment (ogazitt) [inline docs/edge-authorizers/discovery-flow.mdx]: Maybe "<org-name>/<image-name>/blobs/sha256:<digest>" would be a better way to write this?  (assuming I'm interpreting correctly)

File: docs/edge-authorizers/discovery-flow.mdx [hunk 2]
```diff
@@ -128,8 +127,8 @@ The expected response should be:
...
+          "service": "<UUID>-tenant",
```
Comment (ogazitt) [inline docs/edge-authorizers/discovery-flow.mdx]: should this be `"service": "<tenant-id>"` ?

File: docs/edge-authorizers/discovery-flow.mdx [hunk 3]
```diff
@@ -146,14 +145,14 @@ The expected response should be:
         }
       },
       "services": {
-        "aserto-bundler": {
+        "<UUID>-tenant": {
```
Comment (ogazitt) [inline docs/edge-authorizers/discovery-flow.mdx]: same here?

File: docs/edge-authorizers/discovery-flow.mdx (hunk 1 above)
Comment (carabasdaniel) [reply docs/edge-authorizers/discovery-flow.mdx]: resolver, thank you

File: docs/edge-authorizers/discovery-flow.mdx (hunk 2 above)
Comment (carabasdaniel) [reply docs/edge-authorizers/discovery-flow.mdx]: Actually for the service identification we use the registry connection id and the registry org

**Review by rschwabco — APPROVED** [review]

**Pull request merged**